content/en/blog/_posts/2023-12-20-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Cloud Provider Integrations Are Now Separate Components"
+date: 2023-12-20
+slug: cloud-provider-integration-changes
+---
+
+**Authors:** Michael McCune (Red Hat), Andrew Sy Kim (Google)
+
+For Kubernetes v1.29, you need to use additional components to integrate your
+Kubernetes cluster with a cloud infrastructure provider. By default, Kubernetes
+v1.29 does not run with the legacy, compiled-in cloud provider integrations
+enabled.
+
+In 2018, the Kubernetes community agreed to form the Cloud Provider Special
+Interest Group (SIG), with a mission to externalize all cloud provider
```

Comment:
Do we still want to reference this older blog post somewhere? https://kubernetes.io/blog/2019/04/17/the-future-of-cloud-providers-in-kubernetes/

Reply:
i think that's nice background reading, but probably not essential to the new material. that said, i'd like to include a link, it might be easiest to just link in the text of this paragraph.